docs/interop/javascript.md

```diff
@@ -13,29 +13,43 @@ println(s"""libraryDependencies += "dev.zio" %%% "zio" % "${zio.BuildInfo.versio
 println(s"""```""")
 ```
 
-For Java Time API which is not a part of ScalaJS, you might have to add dependency that provides `java.time` package to
-avoid linker errors when using `Clock`. ZIO uses [scala-java-time](https://github.com/cquiroz/scala-java-time) for
-running it's test.
-
 ## Example
 
 Your main function can extend `App` as follows.
 This example uses [scala-js-dom](https://github.com/scala-js/scala-js-dom) to access the DOM; to run the example you
 will need to add that library as a dependency to your `build.sbt`.
 
 ```scala
-import org.scalajs.dom.document
-import zio.{App, IO}
+import org.scalajs.dom.{document, raw}
+import zio._
+import zio.duration._
+import zio.clock._
 
-object MyApp extends App {
+object Main extends App {
 
-  def run(args: List[String]): IO[Nothing, Int] =
```

Comment:
Looks great! I would just specify explicit return types for all these methods so it is easier for someone to follow along.

Reply:
Will do this in the next version when mdoc is working.